### PR TITLE
Fixed issue with QRC file registration introduced in Qt 5.15.

### PR DESCRIPTION
**Pull request by anmacdonald**
Introduced a new qrc path: ":/esri.com/import". This has changed
the invocation args of registerComponents. QML toolkit now must call
"registerComponents."

Fixed issue with NorthArrow image path in widgets.

### CODE REVIEW

File: uitools/register/Esri/ArcGISRuntime/Toolkit/internal/register_cpp.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{
+namespace ArcGISRuntime
+{
+namespace Toolkit
+{
+
+namespace
+{
+
+const QString ESRI_COM_PATH = ":/esri.com/imports";
```
Comment (JamesMBallard) [inline uitools/register/Esri/ArcGISRuntime/Toolkit/internal/register_cpp.cpp:47]: Consider QStringLiteral.

File: uitools/register/Esri/ArcGISRuntime/Toolkit/internal/register_qml.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+namespace Toolkit
+{
+
+namespace
+{
+  const QString ESRI_COM_PATH = ":/esri.com/imports";
```
Comment (JamesMBallard) [inline uitools/register/Esri/ArcGISRuntime/Toolkit/internal/register_qml.cpp:30]: Consider QStringLiteral.

**Review by JamesMBallard — APPROVED** [review]
Some suggestions, but looks good. Do we need to coordinate this with the samples and/or test apps?

**Review by ldanzinger — APPROVED** [review]
LGTM

**Comment by anmacdonald** [comment]
> Do we need to coordinate this with the samples and/or test apps?
@JamesMBallard Yeah we will!

**Pull request merged**